A11y improvements and description of learning_bash (#319)
arcus/education_modules: bash_103_combining_commands/bash_103_combining_commands.md

**Open a bash shell.**
If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.

<div class = "care">
<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>

> It can be stressful to interact with files directly from the command line. Throughout this module we will make sure you know what each command does before asking you to do it, and only ask you to make changes within a practice directory that you will download.

Reviewer: not to be the pickiest person in the world, but this still seems like it will be repetitive to read the second part of this, and then read the info outside of the box about downloading a practice directory. 

Is there a way to totally divorce the two and not mention the downloadable directory at all w/in the care box? e.g., 

```suggestion
It can be stressful to interact with files directly from the command line. Throughout this module we will make sure you know what each command does before asking you to do it.  Like any other computational skill, you'll get more comfortable with it the more you practice!
```

Feel free to totally ignore this, though -- it's just a thought I had, but I think it's also probably fine to just leave it as it currently is! :)

**Open a bash shell.**
If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.

<div class = "care">
<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>

> It can be scary to interact with files directly from the command line. To ensure that we aren't touching any of the important files on your computer, you will be downloading a small directory to experiment with. 

Reviewer: That being said, I'm not convinced this entire thing should be in the care box. Should perhaps just the encouragement/emotional care parts be in the box (so the first sentence, with maybe some sort of follow up/more general reassurance)? And then "To ensure .... to start over if you want" in the standard body of the module since that's more general information about how we're approaching this lesson? 

I could imagine some learners who maybe don't have as much imposter syndrome or anxiety about these things skimming over the care boxes -- the second half of what's in there currently is still information I think it's important for them to have regardless of if they care for the more emotionally-focused half

> It can be scary to interact with files directly from the command line. To ensure that we aren't touching any of the important files on your computer, you will be downloading a small directory to experiment with. 

Author: Thank you for (as always) giving such thoughtful feedback!

I've updated it to incorporate these comments, separating out the care box from the more general information.

If you are using a computer with running iOS (i.e. a Mac) you can use the **Terminal** program. If you are on a computer using Windows, open either **WLS** (Windows Linux Subsytem) or **Git Bash**. If you don't have these programs there are instructions for how to download and set them up in the [Bash / Command Line 101](https://liascript.github.io/course/?https://raw.githubusercontent.com/arcus/education_modules/main/bash_command_line_101/bash_command_line_101.md) module.

<div class = "important">
<b style="color: rgb(var(--color-highlight));">Important note</b><br>

> We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.

Author: Good point, it doesn't exactly fit any of the highlight boxes... I'll update it in all of them.

> We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.

Author: However I want to keep it as an important box for the Bash 102 module, since that is the first time people will be moving stuff around using the command line and they might be a bit scared about screwing around with their own files... What do you think?

> We want to be able to search, move, and rename files during this module, but don't want to do that with your important files. Therefore we will set up a little directory with a few files to experiment with. You can safely delete the whole thing afterwards if you want.

Author: Let's reformulate this text and make it a "care" box instead.

The `learning_bash` folder contains a few different types of files and folders for us to experiment with. The ones we will be using in this module are:

> 

Author: You are absolutely right, and that is a great place for this text!

I was trying to make things consistent across the modules, but this doesn't make sense to have in the same place in each module.

Download the [`learning_bash` directory](https://github.com/arcus/learning_bash) from GitHub. Once you go to the link:

1. Click on the green **Code** drop-down button towards the right-hand side of the screen. 

> 2. Select **Download ZIP**.

Reviewer: ```suggestion
2. Select **Download ZIP** from the options that appear.
```
I like having a bit more of the context, but I don't have a good rationale to back this up, so feel free to do whatever you prefer!

Author: Adopted!

Download the [`learning_bash` directory](https://github.com/arcus/learning_bash) from GitHub. Once you go to the link:

> 1. Click on the green **Code** button. There are two "Code" buttons! The one directly below `arcus/learning_bash` functions more like a "home" button. One line below is a dropdown menu labeled `main` and to the right of `main` are three buttons: `Go to File`, `Add File`, and `Code`. This is the **Code** button you want.

Reviewer: This might be sufficient information instead? Describing where it is on the screen gives more context to individuals who are colorblind, and the fact that the button is a drop-down is additional info that I think would also be useful to those using screenreaders. 

I think otherwise it gets a bit too text heavy and overwhelming, and also we'd need to add even more text to be accurate if we're trying to get exact location to screen readers (ie, they also have to go past "branch" and "tag" links on their way across the row). 

I'm only making this suggestion here, I figure we can workshop it and then once we are fully good with it, change it in all the other places in this PR as well. 

```suggestion
1. Click on the green **Code** drop-down button on the right-hand side of the screen. 
```

Author: I'm going to find a time for us to workshop this together because I think you are totally right that it is too text heavy, but I'm also worried because the **Code** button we want might be in the center of the screen, or even disappear entirely into a `...` menu if the browser window is narrow enough....

Reviewer: change `on` to `towards`

Again there is no output shown because it was redirected into the file `animals.txt`. Check that `animals.txt` now contains the word count at the top, followed by a sorted copy of the data from `Animals.csv`.

> <div class = "behind-the-scenes">

Reviewer: I feel like with some slight tweaking, this actually could be a perfectly acceptable learn more, since we're linking out? see my suggestion below

<b style="color: rgb(var(--color-highlight));">Behind the scenes</b><br>

> In the Bash language, right arrows `>` redirect output, and left arrows `<` redirect input. You are unlikely to need to redirect input since most of the time just entering the input you want, like `Animals.csv`, is sufficient. However if you end up writing more complicated pipelines in the future it may be useful to [learn more about redirecting input](https://www.gnu.org/software/bash/manual/html_node/Redirections.html)

Reviewer: ```suggestion
In the Bash language, right arrows `>` redirect output, and left arrows `<` redirect input. We won't cover redirecting output in this module, however if you end up writing more complicated pipelines in the future it may be useful to [learn more about redirecting input](https://www.gnu.org/software/bash/manual/html_node/Redirections.html)
```

Author: Nice!
